README.md

```diff
@@ -1 +1,11 @@
 # covalent-indexer-go
+
+## Avro schema update
+In case you want to modify the existing avro schema, after you finish your changes, you need to re-generate the corresponding code, by:
+
+1. Get code generator binary
+```bash
+go install github.com/elodina/go-avro/codegen@latest
```

Comment:
go get instead of install

Reply:
instead of `@latest` you can use https://github.com/elodina/go-avro/releases/tag/v0.1.0.0 this release